api/test/integration/env/base/nginx-lb/Dockerfile

```diff
@@ -3,3 +3,5 @@ FROM nginx
 COPY nginx.conf /etc/nginx/nginx.conf
 
 ADD entrypoint.sh /scripts/entrypoint.sh
+
+HEALTHCHECK --interval=30s --timeout=3s CMD curl --fail http://localhost/ || exit 1
```

Comment:
I don't mark it as a requested change because I'm not sure if there is a policy for this, but perhaps we should add an additional line below